src/DonationForms/resources/styles/components/_gateways.scss

```diff
@@ -1,192 +1,96 @@
-        --tab-block-padding: 1.375rem;
-        --tab-block-size: calc(var(--tab-block-padding) * 2 + var(--tab-radio-diameter));
-        --tab-inline-padding: 1.375rem;
-        --tab-transition: 0.2s ease-in-out;
-        position: relative;
-        -webkit-padding-before: var(--tab-block-size);
-        padding-block-start: var(--tab-block-size);
-        background-color: #fafafa;
+        background-color: var(--givewp-shades-white);
         transition: background-color var(--tab-transition);
-        border-radius: 0.25rem;
-        list-style: none;
-        margin: 0;
-    }
-
-    label {
-        color: #444;
-        list-style: none;
-        --tab-radio-diameter: 1.25rem;
-        --tab-radio-border-size: 0.0625rem;
-        --tab-block-padding: 1.375rem;
-        --tab-block-size: calc(var(--tab-block-padding) * 2 + var(--tab-radio-diameter));
-        --tab-inline-padding: 1.375rem;
-        --tab-transition: 0.2s ease-in-out;
-        box-sizing: inherit;
-        cursor: pointer;
-        --font-size: 1em;
-        --line-height: 1.25;
-        --block-size: calc(var(--font-size) * var(--line-height));
-        position: absolute;
-        inset-inline-start: calc(var(--tab-inline-padding) + var(--tab-radio-diameter) + 0.5rem);
-        inset-inline-end: var(--tab-inline-padding);
-        inset-block-start: calc((var(--tab-radio-diameter) - var(--block-size)) / 2 + var(--tab-block-padding));
-        font-weight: bold;
-        font-size: var(--font-size)/var(--line-height);
-        font-family: var(--givewp-primary-color);
-        line-height: var(--line-height);
-    }
-
-    input[type="radio"] {
+        border-radius: var(--givewp-rounded-4);
+        border-color: var(--givewp-grey-400);
         list-style: none;
-        --tab-radio-diameter: 1.25rem;
-        --tab-radio-border-size: 0.0625rem;
-        --tab-block-padding: 1.375rem;
-        --tab-block-size: calc(var(--tab-block-padding) * 2 + var(--tab-radio-diameter));
-        --tab-inline-padding: 1.375rem;
-        --tab-transition: 0.2s ease-in-out;
-        box-sizing: inherit;
-        cursor: pointer;
-        border-start-start-radius: inherit;
-        border-start-end-radius: inherit;
-        border-end-start-radius: inherit;
-        border-end-end-radius: inherit;
-        position: absolute;
-        inset-block-start: 0;
-        inset-inline: 0;
-        block-size: var(--tab-block-size);
-        inline-size: 100%;
         margin: 0;
-        border-width: 0.0625rem;
-        border-style: solid;
-        appearance: none;
-        font-size: 1rem;
-        transition: border-color var(--tab-transition);
-        border-color: #000;
-        z-index: 10;
-        background-color: transparent;
-        box-shadow: none;
 
-        &:hover,
-        &:focus,
-        &:active {
-            --background-color: transparent;
-            border-color: var(--givewp-primary-color);
-            outline: 0;
+        label {
```

Comment:
Pointers on labels isn't actually the default behavior, even though they give the input focus. I think I'd rather keep this as how they naturally work.

Reply:
I agree if this was just a radio input - but we styled the gateways like accordions with the notion that if you click on the label - you are choosing the gateway because it displays the fields.

Reply:
Ohhh, I see what you're saying. I agree: fd968cddf